src/app/core/entity/schema/README.md

```
@@ -28,32 +24,39 @@ The logical flow looks something like this:
-
 `EntitySchemaService` manages a registry of "data types",
 i.e. transformation functions that will be called for a specific schema field's dataType.
 
 Basic data transformations for `string`, `number`, `date` and `month` are supported by default.
 You can register your own types by implementing services extending `DefaultDatatype` and
 providing these through Angular dependency injection using `multi: true`.
-
-Also see: [How to create a new Datatype](../how-to-guides/create-a-new-datatype.html).
+Also see [Create a New Datatype](../../basic-datatypes/README.md).
 
 ### Schema options
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)